wdl-analysis/src/types/v1.rs

```diff
@@ -692,6 +693,33 @@ impl<'a, C: EvaluationContext> ExprTypeEvaluator<'a, C> {
         self.placeholders -= 1;
     }
 
+    fn invalid_placeholder_option(
```

Comment:
We should move this function into `diagnostics.rs`, where we consolidate all diagnostic messages from static analysis.